src/components/reportingCards/ReportingCard8.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReportingCard from './ReportingCard';
+import useLoadData from './ReportingCard.hook';
+import PeopleIcon from '@material-ui/icons/People';
+
+export default function component(props) {
+  const { startDate, endDate, disableSeeMore, rows } = props;
+
+  const data = useLoadData(
+    startDate,
+    endDate,
+    'gender_details',
+    'gender_details',
+    (data) =>
+      data.gender_details.reduce((acc, cur) => acc + Number(cur.number), 0),
+    rows
+  );
+
+  const formatData = (data) => {
+    const result = data.top.map((item) => {
+      return {
+        name:
+          item.name && item.name.match(/f/i)
+            ? 'Female'
+            : item.name
+            ? 'Male'
+            : 'Unknown',
```

Comment:
In the database/API we should support _Female_, _Male_ and _Non-binary_

Reply:
Good call.